### PR TITLE
add more examples (from last A-Frame Feature Friday)

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,12 +108,17 @@ Use React with A-Frame. Share your A-Frame React Components!
...
+- [Cardboard Treasure Hunt Example](https://i3games.github.io/aframe-cardboard-demo/) (by [Michael Straeubig](https://github.com/i3games/)) - An A-Frame implementation of the <a href="https://developers.google.com/cardboard/android/get-started">Google Cardboard “Treasure Hunt” demo</a> [[Code]](https://github.com/i3games/aframe-cardboard-demo/)
+- [Cardboard Dungeon](https://chrismwaite.github.io/cardboard-dungeon/) (by [Christopher Waite](https://twitter.com/chrismwaite/)) - A first-person dungeon crawler designed for mobile [[Code]](https://github.com/chrismwaite/cardboard-dungeon/)
+- [Where in the World Is Carmen Sandiego?](https://ybinstock.github.io/carmensandiego_360/) (by [@ybinstock]) - A scavenger hunt game inspired by the popular series [[Code]](https://github.com/ybinstock/carmensandiego_360/)
+- [Screencast: Using A-Frame with Epson Moverio and Crosswalk](https://www.youtube.com/watch?v=Tt-pX1JMt60) (by [Pablo Mendigochea](https://twitter.com/hololeo/)) - Tech talk on how to build WebVR and VR apps with Crosswalk using A-Frame for the [Epson Moverio](http://www.epson.com/moverio) augmented-reality smart glasses
```
Comment (cvan) [inline README.md]: oops, wrong spot

File: README.md [hunk 2]
```diff
@@ -108,12 +109,16 @@ Use React with A-Frame. Share your A-Frame React Components!
...
 - [threeschwifty](https://ngokevin.github.com/threeschwifty) - 360-degree photo and video viewer [[Code]](https://github.com/ngokevin/threeschwifty)
 - [VR GIF](http://x627.com/vr-gif/) - GIFs in VR [[Post]](http://x627.com/a-frame-is-awesome/)
-- [Virtual Art School Tour](http://stampswebdesign.com/withersc/hud/index.html) (by [@withersc]) - 360-degree photos with navigation
+- [Virtual Art School Tour](http://stampswebdesign.com/withersc/hud/index.html) (by [@withersc]) - Tour of the [Stamps School of Art & Design](http://stamps.umich.edu/), built using 360-degree photos with a navigation carousel
 - [360-Degree Photosphere Viewer](https://www.ideaspacevr.org/examples/aframe/aframe-360-degree-photosphere) - 360-degree photo viewer with navigation [[Code]](https://github.com/IdeaSpaceVR/aframe-360-degree-photosphere)
-- [Space Themed Ecommerce Store](http://vr.lasershop.ca/) (by [@drawvr])
+- [Space-themed Ecommerce Store](http://vr.lasershop.ca/) (by [@drawvr])
```
Comment (ngokevin) [inline README.md]: `E-Commerce` would be nice

File: README.md [hunk 3]
```diff
@@ -108,12 +109,16 @@ Use React with A-Frame. Share your A-Frame React Components!
...
 - [Tunnel Ride](http://drawvr.com/tunnel/) (by [@drawvr])
 - [Wheel Ride](http://drawvr.com/wheel/) (by [@drawvr])
 - [Equalizer Ride](http://drawvr.com/equalizer/) (by [@drawvr])
+- [Cardboard Treasure Hunt Example](https://i3games.github.io/aframe-cardboard-demo/) (by [Michael Straeubig](https://github.com/i3games/)) - An A-Frame implementation of the <a href="https://developers.google.com/cardboard/android/get-started">Google Cardboard “Treasure Hunt” demo</a> [[Code]](https://github.com/i3games/aframe-cardboard-demo/)
```
Comment (ngokevin) [inline README.md]: Can use Markdown for the link

**Comment by ngokevin** [comment]
r+wc good stuff

File: README.md (hunk 3 above)
Comment (cvan) [inline README.md]: oops, forgot about that one

File: README.md (hunk 2 above)
Comment (cvan) [inline README.md]: yeah, I was going to - shall change

**Comment by ngokevin** [comment]
r+

**Comment by cvan** [comment]
rebased

File: README.md [hunk 4]
```diff
@@ -109,14 +110,18 @@ Use React with A-Frame. Share your A-Frame React Components!
...
 - [360-Degree Photosphere Viewer](https://www.ideaspacevr.org/examples/aframe/aframe-360-degree-photosphere) - 360-degree photo viewer with navigation [[Code]](https://github.com/IdeaSpaceVR/aframe-360-degree-photosphere)
-- [Space Themed Ecommerce Store](http://vr.lasershop.ca/) (by [@drawvr])
+- [Space-themed E-commerce Store](http://vr.lasershop.ca/) (by [@drawvr])
 - [Tunnel Ride](http://drawvr.com/tunnel/) (by [@drawvr])
 - [Wheel Ride](http://drawvr.com/wheel/) (by [@drawvr])
 - [Equalizer Ride](http://drawvr.com/equalizer/) (by [@drawvr])
 - [Test Scape](http://x627.com/testscape/) (by [Michael Verdi](http://x627.com/directory/))
 - [INFIVERSE](http://infiverse.com/) (by [@drawvr]) Navigate a collection of cool A-Frame sites by walking through rooms and doorways (by [@drawvr])
+- [Cardboard Treasure Hunt Example](https://i3games.github.io/aframe-cardboard-demo/) (by [Michael Straeubig](https://github.com/i3games/)) - An A-Frame implementation of the [Google Cardboard "Treasure Hunt" demo](https://developers.google.com/cardboard/android/get-started">) [[Code]](https://github.com/i3games/aframe-cardboard-demo/)
```
Comment (cvan) [inline README.md]: remove "Example"

**Comment by cvan** [comment]
r+

**Pull request merged**